webapp/channels/src/packages/mattermost-redux/src/selectors/entities/scheduled_posts.ts

```diff
@@ -63,7 +63,7 @@ export function showChannelOrThreadScheduledPostIndicator(state: GlobalState, ch
     const allChannelScheduledPosts = state.entities.scheduledPosts.byChannelOrThreadId[channelOrThreadId] || emptyList;
     const eligibleScheduledPosts = allChannelScheduledPosts.filter((scheduledPostId: string) => {
         const scheduledPost = state.entities.scheduledPosts.byId[scheduledPostId];
```

Comment:
0/5 To avoid future errors, you can make this record to be `{[x: string]: ScheduledPost | undefined}`. That way Typescript will yet at you if you don't check presence.

Reply:
Done!